lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/RestBasedAccessControlConfig.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.base.security;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+import javax.validation.constraints.NotNull;
+
+public class RestBasedAccessControlConfig
+        extends BaseAccessControlConfig
+{
+    public static final String SECURITY_JSON_POINTER = "security.json-pointer";
+
+    private String restUrl;
+    private String jsonPointer = "";
+
+    @NotNull
+    public String getRestUrl()
+    {
+        return restUrl;
+    }
+
+    @Config(SECURITY_CONFIG_FILE)
```

Comment:
Nit: What do you think - if we have a different config name here ? One advantage is that - when we search for the config property in codebase, it allows us to identify which config it belongs to.

Reply:
good idea - done